public/app/features/auth-config/index.ts

```diff
@@ -0,0 +1,62 @@
+import { AuthProviderStatus, Settings, SettingsSection } from 'app/types';
+
+import { AuthProviderInfo, GetStatusHook } from './types';
+
+export * from './types';
+
+const registeredAuthProviders: AuthProviderInfo[] = [
+  // { id: 'azuread', displayName: 'Azure AD OAuth2' },
+  // { id: 'generic_oauth', displayName: 'OAuht2' },
+  // { id: 'github', displayName: 'GitHub OAuth2' },
+  // { id: 'gitlab', displayName: 'GitLab OAuth2' },
+  // { id: 'google', displayName: 'Google OAuth2' },
+  // { id: 'grafana_com', displayName: 'grafana.com OAuth2' },
+  // { id: 'jwt', displayName: 'JWT' },
+  // { id: 'okta', displayName: 'Okta OAuht2' },
+];
+
+const authProvidersConfigHooks: { [k: string]: GetStatusHook } = {};
```

Comment:
```suggestion
const authProvidersConfigHooks: Record<string, GetStatusHook> = {};
```
I think this'd work and might be easier to understand.